lms/djangoapps/oauth_dispatch/tests/test_views.py

```diff
@@ -318,3 +318,81 @@ def test_get_view_for_dop(self):
     def test_get_view_for_no_backend(self):
         view_object = views.AccessTokenView()
         self.assertRaises(KeyError, view_object.get_view_for_backend, None)
+
+
+class TestRevokeTokenView(_DispatchingViewTestCase):  # pylint: disable=abstract-method
+    """
+    Test class for RevokeTokenView
+    """
+
+    login_with_access_token_url = reverse("login_with_access_token")
+    revoke_token_url = reverse('revoke_token')
+    access_token_url = reverse('access_token')
+
+    def setUp(self):
+        super(TestRevokeTokenView, self).setUp()
+        response = self.client.post(self.access_token_url, self.access_token_post_body())
+        access_token_data = json.loads(response.content)
+        self.access_token = access_token_data['access_token']
+        self.refresh_token = access_token_data['refresh_token']
+
+    def access_token_post_body(self, refresh_token=None):
```

Comment:
Nit: Create separate helper methods for each.  For example: `_post_body_with_password` and `_post_body_with_refresh_token`.  This follows the best practice described in the Clean Code book and makes it easier to understand the code.